api/api_full.go

```diff
@@ -146,6 +146,9 @@ type FullNode interface {
 	// MpoolPending returns pending mempool messages.
 	MpoolPending(context.Context, types.TipSetKey) ([]*types.SignedMessage, error)
 
+	// MpoolSelect returns a list of pending messages for inclusion in the next block
+	MpoolSelect(context.Context, types.TipSetKey) ([]*types.SignedMessage, error)
```

Comment:
Can we add a list of 'priority addresses' to this API? And have it prioritize messages from those addresses, even if they arent necessarily the most profitable ones?

Reply:
can be done as a followup, but i think its a pretty important thing to have

Reply:
yes, we can do that, but I was thinking of doing it in a subsequent pr after discussing it.

Reply:
agreed on importance, let's discuss on the colo -- I want to think about it a bit.

Reply:
I think we want a separate api call because we need to know priority addresses for pruning too -- we shouldn't prune messages from our miner etc.